File: src/main/java/io/github/apollointhehouse/mixin/WorldAccessor.java
package io.github.apollointhehouse.mixin;

import net.minecraft.core.world.World;
import net.minecraft.core.world.save.LevelData;
import org.spongepowered.asm.mixin.Mixin;
import org.spongepowered.asm.mixin.gen.Accessor;

@Mixin(value = World.class, remap = false)
public interface WorldAccessor {
	@Accessor("levelData")
	LevelData getLevelData();
}

File: src/main/kotlin/io/github/apollointhehouse/mixin/WorldAccessor.kt
package io.github.apollointhehouse.mixin

import net.minecraft.core.world.World
import net.minecraft.core.world.save.LevelData
import org.spongepowered.asm.mixin.Mixin
import org.spongepowered.asm.mixin.gen.Accessor

@Mixin(value = [World::class], remap = false)
interface WorldAccessor {
	@get:Accessor("levelData")
	val levelData: LevelData
}
